test/classes/delete-free/nil-checking/nilcheck-emptied.chpl

```diff
@@ -51,7 +51,7 @@ proc bad4() {
   var x = new owned MyClass(1);
   ref y = x;
   var z = x;
-  y.clear();
+  y = nil;
```

Comment:
Can you try prepending `writeln(y);` onto Line 54?

The reason is that `y.clear()` tested the dead value detection through a ref. That does not happen any longer upon `y = nil;`. So using `y` explicitly will hopefully restore that. I suggest putting it on the same line so that .good does not need to change.